ignite/contrib/engines/tbptt.py

```diff
@@ -73,7 +84,7 @@ def create_supervised_tbptt_trainer(
 
     """
 
-    def _update(engine, batch):
+    def _update(engine: Engine, batch: Tuple[torch.Tensor, torch.Tensor]):
```

Comment:
Let's use `Sequence[torch.Tensor]` for batch

Reply:
Done